src/app/components/navbar.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+import Image from "next/image";
+
+export default function NavBar() {
+  return (
+    <>
+      <div className="row nav--bar">
+        <Image src="/LogoPunctoranj.svg" width={50} height={50} alt="logo" />
+        <div className="logo"></div>
+        <Link href="/about" style={{ marginLeft: "auto" }} className="btn">
```

Comment:
same here. let's avoid inline styling where possible or at lest have it defined outside

Reply:
Totally agree. Here I just needed a `marginLeft:auto` so I can push a specific child of the `flex` container to the right.  I don't like having a ` css file ` with classes that do on thing only.